app/assets/stylesheets/top-bar.scss

```diff
@@ -311,6 +311,13 @@
     padding: 8px;
     font-size: 0.9em;
     background: #e8e7e7;
```

Comment:
```suggestion
```
I believe you intended to replace this line with the 1 line below.

Reply:
That is what I did with the earlier ones but I realised that broke things for IE, leaving that line in means IE will still get the original colours

Reply:
Ah I see. Please disregard that then.

Reply:
Unless we don't care about IE, I doubt we get much traffic from IE?

Reply:
@benhalpern , what's your thought on this?

Reply:
I'm forming the habit of answering questions in the form of documentation updates.

I didn't realize our public docs didn't have browser support listed anywhere.

So... we do not officially support any version of IE.

https://github.com/thepracticaldev/dev.to/pull/1378/files

Reply:
In the vain of not going "out of our way to break anything" should we leave in the initial colours? I'll have to go back and add them to the other 2 commits, bit that's not too much work.

Reply:
In the vain of not going "out of our way to break anything" should we leave in the initial colours? I'll have to go back and add them to the other 2 commits, bit that's not too much work.